PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,111 @@
+> Reference: [PR Submission Guide](/submission_process.md)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+### Dependencies
+> List all libraries and imports utilized, e.g. Ownable.sol from OZ
+
+### Externalities
+> List all external (to this PR) contracts that are interacted with - within Yield ecosystem and without.
+
+> _Note to Reviewer: Examine every component the feature may touch. Then go back through excluded components and ensure they cannot be affected._
+
+### Risk 
+> Highlight areas of risk. E.g. Assembly code blocks, use of unchecked, function call made to an external contract that lies outside of the Yield ecosystem. 
+
+> Example: fn A in contractB.sol makes a call to someexternalcontract.sol
+
+## Contracts
+> This section serves as a brief for the reviewer, code author is to introduce contracts and elaborate on salient points.
+
+> ContractA.sol -> description + points
+
+> ContractB.sol -> description + points
+
+> Reference: See section '[New contracts and areas of risk](https://github.com/yieldprotocol/yieldspace-tv/pull/3)'
+
+### Complexity
+> Highlight areas of abstraction/complexity; e.g. functions that have complicated math or are deeply layered.
+
+> Example: ContractA.sol, Function doSomeComplexMath()
+
+## Testing
+> Elaborate on your testing approach - assumptions made, concerns and any caveats.
+
+> Map your state inheritance testing flow. This could be in the form of a [text tree](https://github.com/calnix/Collateralized-Vault/blob/AddedCode/Readme.md#testing) or an illustrated diagram (excalidraw/draw.io).
+
+> Negative tests, Positive tests, Adversarial and Transition tests should be listed out in each state node. Highlight both fuzzing and invariant tests.
+
+> WHY: Offers a high-level risk surface of the testing coverage. Implementer can be certain they have not missed any possible states. Auditor quickly appreciate what has been tested, and spot any gaps.  
+
+**Example:**_(Please delete and replace)_
+
+```
+(StateZeroTest) 
+(actions available: deposit, withdraw, redeem)
+- negative tests
+- adversarial tests
+- positive tests 
+- transition test
+
+(StateSomeStateTest)
+(actions available)
+- etc,...
+```
+
+### Fuzz testing
+> Highlight key tests/points regarding fuzz testing done
+
+### Highlight Invariant tests
+> Highlight key tests/points regarding invariant testing done
+
+### Highlight Economic tests
+> Highlight key tests/points regarding economics testing done
+
+### Highlight Integration tests
+> Highlight key tests/points regarding integration testing done
+
+
+## Reports
+> Insert report outputs here
```

Comment:
idk how i feel about this.  we include gas reports by default now.  non one uses SMTChecker and I don't see anyone using it any time soon.  Slither is nice but, accept for me, no one is currently using it (also the reports are pretty big)

Maybe delete this section and lets have a catchall comment at the end that says "Please also include anything else relevant to this pr such as Slither reports..."